packages/protocol/src/index.ts

```diff
@@ -13,31 +13,6 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
-import { SetBoundsAction, SetViewportAction } from 'sprotty-protocol/lib/actions';
-import { Action } from './action-protocol';
-import { hasBooleanProp, hasObjectProp, hasStringProp } from './utils/type-util';
-
-// Add the is() function to the namespace declarations of sprotty-protocol actions
-declare module 'sprotty-protocol/lib/actions' {
-    // eslint-disable-next-line @typescript-eslint/no-shadow
-    namespace SetViewportAction {
-        export function is(object: any): object is SetViewportAction;
-    }
-
-    // eslint-disable-next-line @typescript-eslint/no-shadow
-    namespace SetBoundsAction {
-        export function is(object: any): object is SetBoundsAction;
-    }
-}
-
-SetViewportAction.is = (object: any): object is SetViewportAction =>
-    Action.hasKind(object, SetViewportAction.KIND) &&
-    hasStringProp(object, 'elementId') &&
-    hasObjectProp(object, 'newViewport') &&
-    hasBooleanProp(object, 'animate');
-
-SetBoundsAction.is = (object: any): object is SetBoundsAction =>
-    Action.hasKind(object, SetBoundsAction.KIND) && hasObjectProp(object, 'bounds');
 
 // Partial reexport of sprotty-protocol
 export { Viewport } from 'sprotty-protocol/lib/model';
```

Comment:
I believe this file also exports `SModelIndex`, one of the few classes still being exported with the `SModel`-prefix. Similar to `SModelFactory` and `SModelRegistry`.

Reply:
Good catch. We don`t need `SModelIndex` from protocol at all. This is an index of the serialize format (schema) which we don't use.
Regarding the other `SModel`-prefix exports. These are on purpose
- SEdgeImpl, SGraphImpl, SModelRegistry and SModelFactory are exported without alias because we have dedicated `G` subclasses for them in the client.

I have added inline comments to make this clearer.

Reply:
I see, I was just surprised that the code completion in the client package offered me those types instead of the expected G types but if they are only defined in the client, that makes sense. Thank you for clarifying!